tests/config_test.go

```diff
@@ -112,7 +112,12 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				expectedOutput := "value1value2value3"
 
 				By("Running VMI")
-				vmi := libvmifact.NewAlpine(libvmi.WithConfigMapDisk(configMapName, configMapName))
+				vmi := libvmifact.NewAlpineWithTestTooling(libvmi.WithConfigMapDisk(configMapName, configMapName))
```

Comment:
I noticed that the image utilized for this test has also been modified. Please ensure the commit message is updated to provide the necessary reasoning for this change.

Reply:
Done